Upstream-Status: 2012/05/02

automake 1.12 has depricated automatic de-ANSI-fication.

fix this issue:
| configure.ac:564: error: automatic de-ANSI-fication support has been removed

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/05/02
Index: rpm-5.4.0/configure.ac
===================================================================
--- rpm-5.4.0.orig/configure.ac
+++ rpm-5.4.0/configure.ac
@@ -561,7 +561,6 @@ dnl # platform specific checks
 AC_SEARCH_LIBS([strerror],[cposix])
 
 dnl # standard checks
-AM_C_PROTOTYPES
 AC_C_INLINE
 AC_C_BIGENDIAN
 
